### DISCUSSION TITLE
Set nodata according to a binary mask

### DISCUSSION THREAD

**Discussion by remi-braun** [corteva/rioxarray#311] [Q&A]
Is there any function that sets the pixels to the nodata according to a binary mask ?
Or simply doing 
```python
# xda: xr DataArray
# nodata: np.ndarray (bool) with the same shape
xda[np.where(nodata)] = np.nan
```
works without forgetting anything ?

**Comment by snowman2** [comment]
xarray also has `where` that you could use. http://xarray.pydata.org/en/stable/generated/xarray.where.html
